scripts/prepare/bundle.ts

```diff
@@ -134,11 +138,19 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
 
 /* UTILS */
 
-async function getDTSConfigs({ formats, entries }: { formats: Formats[]; entries: string[] }) {
+async function getDTSConfigs({
+  formats,
+  entries,
+  optimized,
+}: {
+  formats: Formats[];
+  entries: string[];
+  optimized: boolean;
+}) {
   const tsConfigPath = join(cwd, 'tsconfig.json');
   const tsConfigExists = await fs.pathExists(tsConfigPath);
 
-  const dtsBuild = formats[0] && tsConfigExists ? formats[0] : undefined;
+  const dtsBuild = optimized && formats[0] && tsConfigExists ? formats[0] : undefined;
```

Comment:
turned out we were always generating the definitions :(
oops.

Reply:
This might make local builds a bit faster again. Though this change hasn't been around for long.